src/land/afwa_snod2ioda.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+import pygrib
+import time, os, sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+from pathlib import Path
+
+IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+if not IODA_CONV_PATH.is_dir():
+    IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
+sys.path.append(str(IODA_CONV_PATH.resolve()))
+
+import ioda_conv_ncio as iconv
+from orddicts import DefaultOrderedDict
+
+vName = {
+    'A': "afwa_snow_depth",
+}
+
```

Comment:
Agreed, this is very important to include, otherwise how can someone use the observations properly if we don't know the units.

Reply:
> I urge the explicit assignment of an attribute of units to the variable. Maybe I missed it, but I don't think I see it being done. I assume it is kg per m-squared, which is equivalent to mm.

I add AttrData["afwa_snowDepth_units"] = "mm" as suggested.

Reply:
Here is the current draft of my document for [IODA variable naming convention](https://docs.google.com/spreadsheets/d/1EQmJdkqsL3u2-SdpKSA4gkDLd6fCt3iAQ-XNcjz_2M0).  It may require that I give you permission to view. If that happens, just let me know.  Notice there are a series of sheets/tabs at the bottom and I made categories such as "MetaData," "ObservedVariables," "DerivedVariables," and DA-related-variables.  You should not name (in my opinion) your variable as "afwa_snow_depth."  The point is that the variable itself should be simply "snowDepth" (row=63 in my sheet) and you can use a MetaData variable such as "dataProviderOrigin" (row=12) set to AFWA (or probably better yet "Air Force Weather Agency."  The point is for another snow depth analysis (example=NESDIS), then their variable name is the same but their MetaData describes their source.  Lastly, the assignment of a variable's units is not done the way you describe.  See an existing netCDF with CF convention for how it is done properly please.

Reply:
I changed variable name as snowDepth as suggested. I add the variable units with CF convention.  I also removed "Air Force Weather Agency AFWA" as the other ioda-converters' codes did.  Please take re-review and see if it is kay. Thank you.